js/src/wp-seo-replacevar-plugin.js

```diff
@@ -529,5 +529,14 @@ var ReplaceVar = require( "./values/replaceVar" );
 		return parentText;
 	};
 
+	/*
+	 * STATIC VARIABLES
+	 */
+
+	/**
+	 * Exposes the ReplaceVar class for functionality of plugins integrating.
```

Comment:
As this is a single line comment, it should be formatted as such:
`// Exposes the ReplaceVar class for functionality of plugins integrating.`